### ISSUE TITLE
Replace usage of twisted.python.text in twisted.words

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6544]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#6544|
|Type|task|
|Created|2013-06-03 14:22:33Z|
|Branch|https://github.com/twisted/twisted/tree/replace-text-words-6544|

twisted.python.text is being deprecated ([#6341](https://github.com/twisted/twisted/issues/6341)) and its usage should be replaced in twisted.words.

<details><summary>Searchable metadata</summary>

```
trac-id__6544 6544
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_replace_text_words_6544 branches/replace-text-words-6544
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__words words
keywords__ 
time__1370269353000000 1370269353000000
changetime__1372301018000000 1372301018000000
version__None None
owner__thijs thijs
cc__thijs cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra removed owner|
|-|-|

Replaced the usage in twisted.words and added tests for uncovered lines.

Forced a [build](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/replace-text-words-6544).

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_3"></a>|@tomprince set owner to @thijstriemstra|
|-|-|

This looks good. Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38929]) Merge replace-text-words-6544: Replace the usage of twisted.python.text because it will be deprecated.

Author: thijs
Reviewer: tom.prince
Fixes: [#6544](https://github.com/twisted/twisted/issues/6544)
